### PR TITLE
[ticket/16822] Replace patchwork/utf8 with symfony/polyfill

### PR DESCRIPTION
**Pull request by rxu**
3.3.x version.

Checklist:

- [x] Correct branch: master for new features; 3.3.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

<a href="https://tracker.phpbb.com/browse/PHPBB3-16822">PHPBB3-16822</a>.

### CODE REVIEW

**Review by CHItA — APPROVED** [review]

**Comment by CHItA** [comment]
Can this break any extensions? I assume no one should rely on the specific polyfill we use, but that could be something this might cause problems with. In which case this should go against master. Otherwise LGTM.

**Comment by rxu** [comment]
Infact this just removes `patchwork/utf8`, all mentioned polyfill packages are already present in phpBB. Current versions are 1.22.x so just minorily upgraded to 1.23.x I guess.

**Comment by rxu** [comment]
`master` version: #6251.

File: phpBB/composer.json [hunk 1]
```diff
@@ -45,6 +44,9 @@
 		"symfony/finder": "~3.4",
 		"symfony/http-foundation": "~3.4",
 		"symfony/http-kernel": "~3.4",
+		"symfony/polyfill-intl-normalizer": "^1.23",
+		"symfony/polyfill-mbstring": "^1.23",
+		"symfony/polyfill-php72": "^1.23",
```
Comment (3D-I) [inline phpBB/composer.json:49]: Is this just related to PHP 7.2 or greater?

Comment (rxu) [reply phpBB/composer.json:49]: Nope, this polyfill just reimplements utf related functions from PHP 7.2 core.

**Pull request merged**